### DISCUSSION TITLE
Add kicad_auto container 'flavor' and 'tag' to env variable

### DISCUSSION THREAD

**Discussion by hpopols** [INTI-CMNB/KiBot#527] [Ideas]
It could be nice to have the version & flavor of the kicad_auto image used in an environment variable to be later used in a worksheet and other outputs.

The idea is the same as the kicad version displayed in the default worksheet but also would help to promote/cite this great software.

What do you think?

**Comment by set-soft** [comment]
Looks like a good idea.
I'm doing some experiments with the *dev* images.
I found that for the tag is much easier to use a file, I'm using */etc/kiauto_tag*
For the full/basic flavor I'm defining the *KICAD_AUTO_FULL* environment variable, full images have it defined as 1, not defined for basic images.

**Reply by set-soft** [reply]
Ok, and now should be propagated to all images ...
